dds/DCPS/security/AuthenticationBuiltInImpl.h

```diff
@@ -36,6 +36,14 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 namespace OpenDDS {
 namespace Security {
 
+const std::string Identity_Status_Token_Class_Id("DDS:Auth:PKI-DH:1.0");
+const std::string Auth_Peer_Cred_Token_Class_Id("DDS:Auth:PKI-DH:1.0");
+
+const std::string dds_cert_sn("dds.cert.sn");
+const std::string dds_cert_algo("dds.cert.algo");
+
+const std::string dds_ca_sn("dds.ca.sn");
+const std::string dds_ca_algo("dds.ca.algo");
```

Comment:
For all of these.... no need for `std::string` namespace-scoped constants, especially want to keep them out of headers.
```
const char Identity_Status_Token_Class_Id[] = "DDS:Auth:PKI-DH:1.0";
```